File: 백준/Silver/2563. 색종이/색종이.py
a = int(input())
arr = []
for i in range(100):
  semi_arr = []
  for j in range(100):
    semi_arr.append(0)
  
  arr.append(semi_arr)


for k in range(a):
  x, y = map(int, input().split())
  for i in range(10):
    for j in range(10):
      arr[x + i][y + j] = 1

result = 0

for i in range(100):
  for j in range(100):
    if arr[i][j] == 1:
      result += 1

print(result)

File: 백준/Silver/2563. 색종이/색종이.c
#include <stdio.h>

int main(void) {  
   int map[100][100] = {0};
   int a, x, y;
   scanf("%d", &a);

   for(int i = 0; i < a; i++){
      scanf("%d %d", &x, &y);
      for(int j = 0; j < 10; j++){
         for(int k = 0; k < 10; k++){
            map[x + j][y + k] = 1;
         }
      }
   }

   int result = 0;
   for(int i = 0; i < 100; i++){
      for(int j = 0; j < 100; j++){
         if(map[i][j] == 1){
            result++;
         }
      }
   }

   printf("%d", result);
   
   return 0;
}